common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java

```diff
@@ -67,7 +67,7 @@ public enum SpanName {
     SecretKeyWrapping,
     WrappedKeyAlgorithmIdentifier,
     ProcessWebCpRedirects,
-
     ProvisionResourceAccount,
-    ProcessWebsiteRequest
+    ProcessWebsiteRequest,
+    GetAllSsoTokens
```

Comment:
How often will this be invoked?

Reply:
Discussed offline; this is going to be called multiple times per login session. So I will set a lower sampling rate, and once I get a better idea of an average amount of calls, I'll be able to control that rate via ECS.